backends/tc/tcExterns.h

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (C) 2023 Intel Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions
+and limitations under the License.
```

Comment:
Is it the right license? Just to confirm as MEV TS had different (short) text.
Copyright year shall be updated.

Reply:
This is the license used across other open source backends and tc backend as well. Updated the year.